sw/device/tests/crypto/BUILD

```diff
@@ -8,6 +8,20 @@ load("@ot_python_deps//:requirements.bzl", "requirement")
 
 package(default_visibility = ["//visibility:public"])
 
+opentitan_functest(
+    name = "aes_gcm_functest",
+    srcs = ["aes_gcm_functest.c"],
+    verilator = verilator_params(
+        timeout = "long",
```

Comment:
The test timed out on my FPGA. I specified `cw310  = cw310_params(timeout="long")` and it passed in 179.6s. I think you could get away with "moderate" per https://bazel.build/reference/test-encyclopedia .